LogicalTypes.md

```diff
@@ -366,3 +366,7 @@ optional group my_map (MAP_KEY_VALUE) {
 }
 ```
 
+## Null
+Sometimes when discovering the schema of existing data values are always null and there's no type information.
```

Comment:
This is correct. This is why arrow has a null type in the first place.
I'm wondering what is the impact of adding a primitive type.
The fact that the value is always null simplifies a bit.